src/components/Popup/Popup.tsx

```diff
@@ -96,47 +108,72 @@ export default class Popup extends UIComponent<Extendable<IPopupProps>, IPopupSt
     as: Portal,
     align: 'start',
     position: 'above',
+    accessibility: PopupBehavior as Accessibility,
   }
 
-  public state = { triggerRef: undefined }
+  public state = { triggerRef: undefined, popupOpened: false }
+
+  actionHandlers: AccessibilityActionHandlers = {
+    open: e => this.openPopup(e),
```

Comment:
no need to create other functions here, they already are:
```typescript
open: this.openPopup,
close: this.closePopup,
closeAndFocusTrigger: this.closePopup.bind(this, this.focusTrigger),
```

Reply:
```.bind``` works in a not very straightforward way, so I would need to change function signature, by replacing order of params. Which confuses while reading the code. Also ```this.focusTrigger``` is an arrow function now, in a case to use it as you proposed, I would need either move it above the ```actionHandlers``` or make it a regular function and bind context in the constructor. 
It's kind of tradeoff. If you don't have serious concerns regarding it, I would leave as it is.
Also glad to hear other opinions on it.

Reply:
It is protected, inherited from UIComponent.